modules/k3s/master/policy.yaml

```diff
@@ -80,7 +80,7 @@ metadata:
 spec:
   podSelector:
     matchLabels:
-      k8s-app: metrics-server
+      k8s-app: metrics-master
```

Comment:
was this change intentional - or was this a part of mass replace of 'server' back to 'master' ?

Reply:
nope tks